aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -0,0 +1,174 @@
+---
+title: ASP.NET Core metrics
+description: Metrics for ASP.NET Core apps
+author: rick-anderson
+ms.author: riande
+monikerRange: '>= aspnetcore-8.0'
+ms.date: 6/30/2023
+ms.topic: article
+ms.prod: aspnet-core
+uid: log-mon/metrics/metrics
+---
+
+# ASP.NET Core metrics
```

Comment:
> ASP.NET Core metrics [](http://example.com/codeflow?start=1&length=21)

would it be a good idea to reference [https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics) here?